src/test/java/org/kohsuke/github/AbstractGitHubWireMockTest.java

```diff
@@ -244,6 +260,20 @@ protected void kohsuke() {
         // assumeTrue(login.equals("kohsuke") || login.equals("kohsuke2"));
     }
 
+    private GHCreateRepositoryBuilder getCreateBuilder(String name, boolean useOrg) throws IOException {
+        GitHub github = getGitHubBeforeAfter();
+
+        if (useOrg) {
```

Comment:
Instead of having this be a parameter that gets passed down through the call stack, this should be a setting like `mockGitHub.isUseProxy()` but on `AbstractGitHubWireMockTest`, something like `isTestWithUserAccount()`. This will let users create, debug, and record tests using their account, but also easily turn it off and re-record from the test account later.

Reply:
i feel like i followed this from some other example but i can see about addressing this.